Front-end/src/components/sub-component/Comment.jsx

```diff
@@ -0,0 +1,54 @@
+import {
+  CommentText,
+  Icon,
+  CommentMetadata,
+  CommentGroup,
+  CommentContent,
+  CommentAvatar,
+  CommentAuthor,
+  Form,
+  Comment,
+} from "semantic-ui-react";
+import { useState, useEffect } from "react";
+import moment from "moment";
+
+const Comments = (props) => {
+  const [date, setDate] = useState();
+  const [comment, setComment] = useState([]);
+
+  function CommentAuthors() {
+    fetch(`http://localhost:3000/commentUser/${props.author}`, {
+      method: "GET",
+      headers: {
+        "Content-Type": "Application/json",
+      },
+    }).then((data) =>
+      data.json().then((data) => {
+        console.log(data);
```

Comment:
remove console.log ?

Reply:
In the main branch, it is not there in Comment.js. I most likely noticed it and updated it as soon as possible last week. Closing this conversation now.